blueoil/blueoil_init.py

```diff
@@ -187,15 +190,18 @@ def image_size_filter(raw):
     return image_size
 
 
-def save_config(blueoil_config):
-    env = Environment(loader=FileSystemLoader('./blueoil/templates', encoding='utf8'))
+def save_config(blueoil_config, output=None):
+    env = Environment(loader=FileSystemLoader(TEMPLATE_DIR, encoding='utf8'))
     tpl = env.get_template('blueoil-config.tpl.yml')
 
     applied = tpl.render(blueoil_config)
-    config_file = blueoil_config['model_name'] + ".yml"
-    with open(config_file, 'w') as fp:
+
+    if not output:
```

Comment:
How about checking format of `output` here, instead of just checking this is `None` or not? Such as:
```python
# this is sudo code...may cannot work
if not output or output.endswith('.yml') == False:
```

Reply:
@djKooks 
I looked at the loading process of the config file and I did not specify extensions, so I thought that it was not necessary at init time.

https://github.com/blue-oil/blueoil/blob/4e6abd97f5f740af89484c08a7d69934e58c2d56/blueoil/generate_lmnet_config.py#L77-L96